### DISCUSSION TITLE
Support for static sites

### DISCUSSION THREAD

**Discussion by collimarco** [buildpacks/community#76] [Ideas]
Have you ever thought about using the same techniques for static sites?

It would be interesting to have a buildpack that detects the static framework (like Hugo or Jekyll), compiles the project and runs nginx (or similar) in the docker image in order to serve the requests.

I see a lot of potential.

Do you know if there are any similar buildpacks?

**Comment by dfreilich** [comment]
The Cloud Foundry buildpacks had a staticfile buildpack, which assumed that the sites were compiled, and then used nginx to serve them. We wouldn't provide them, but it could be that projects like Paketo or Heroku could look into creating a hugo/jekyll framework, if they haven't yet.

**Comment by kvedurmu** [comment]
@collimarco 👋  Paketo actually has a [Staticfile Buildpack](https://github.com/paketo-community/staticfile) that we'd love feedback on!  This buildpack supports building of static content apps running on NGINX. We don't have static framework support yet (like Hugo) but the project has been thinking about it. 

@fg-j (a contributor to the Paketo project) also recently built a [hugo buildpack](https://github.com/fg-j/hugo-buildpack) and I bet they'd like feedback on this as well 😄
